### PR TITLE
Add CAN ID filtering example

### PR DESCRIPTION
**Pull request by aaknitt**

### CODE REVIEW

File: doc/bus.rst [hunk 1]
```diff
@@ -59,6 +59,14 @@ Filtering
 Message filtering can be set up for each bus. Where the interface supports it, this is carried
 out in the hardware or kernel layer - not in Python.
 
+Example defining two filters, one to pass 11-bit ID 0x451, the other to pass 29-bit ID 0xA0000:
+
+.. code-block:: python
+
+    filters = [{"can_id": 0x451, "can_mask":0x7FF,"extended":False},{"can_id": 0xA0000, "can_mask":0x1FFFFFFF,"extended":True}]
+    bus = can.interface.Bus(channel='can0', bustype='socketcan_native',can_filters=filters)
```
Comment (pierreluctg) [inline doc/bus.rst]: `socketcan_native` and `socketcan_ctypes` are deprecated, please see https://python-can.readthedocs.io/en/master/interfaces/socketcan.html.
Please use `socketcan` interface.

File: doc/bus.rst [hunk 2]
```diff
@@ -59,6 +59,14 @@ Filtering
 Message filtering can be set up for each bus. Where the interface supports it, this is carried
 out in the hardware or kernel layer - not in Python.
 
+Example defining two filters, one to pass 11-bit ID 0x451, the other to pass 29-bit ID 0xA0000:
+
+.. code-block:: python
+
+    filters = [{"can_id": 0x451, "can_mask":0x7FF,"extended":False},{"can_id": 0xA0000, "can_mask":0x1FFFFFFF,"extended":True}]
```
Comment (pierreluctg) [inline doc/bus.rst]: Please make sure that the code conforms with pep8 and black formatting.
See https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

File: doc/bus.rst [hunk 3]
```diff
@@ -59,6 +59,17 @@ Filtering
 Message filtering can be set up for each bus. Where the interface supports it, this is carried
 out in the hardware or kernel layer - not in Python.
 
+Example defining two filters, one to pass 11-bit ID 0x451, the other to pass 29-bit ID 0xA0000:
```
Comment (pierreluctg) [inline doc/bus.rst:62]: Can we consolidate with https://github.com/hardbyte/python-can/blob/develop/can/bus.py#L330?

**Comment by hardbyte** [comment]
Oh I hadn't noticed this was raised against the master branch. I'll just push the small change to develop. cheers.

**Pull request closed without merging**